utilities/load-test/loadtest/loadtest_test.go

```diff
@@ -413,3 +431,102 @@ func TestAllApplication(t *testing.T) {
 	}
 	t.Log("Prometheus, along with 20 guestbook applications are created, deployed and deleted successfully!")
 }
+
+func TestNResource(t *testing.T) {
+	// TEST 1: Deply 100 guestbook in different namespace
+	t.Log("Testing for 100 applications that uses lightweight non-Pod-based resources\n")
+
+	t.Log("\n\nPods Memory Resource before creation:\n\n")
+	memoryInit := GetPodInfo(kubeconfig, masterUrl, namespace, podName)
+	PodInfoParse(memoryInit)
+
+	// Get Pod Restart Count
+	PodC := PodRestartCount(namespace)
+
+	for i := 1; i < 101; i++ {
+		applicationTest := &appv1.Application{
+			TypeMeta: v1.TypeMeta{
+				Kind:       "Application",
+				APIVersion: "argoproj.io/v1alpha1",
+			},
+			ObjectMeta: v1.ObjectMeta{
+				Name:      "test-app-" + fmt.Sprintf("%d", i),
+				Namespace: "argocd",
+			},
+			Spec: appv1.ApplicationSpec{
+				Source: appv1.ApplicationSource{
+					Path:           "sample-app",
+					TargetRevision: "HEAD",
+					RepoURL:        "https://github.com/samyak-jn/gitops-service-sample-k8s-resources.git",
```

Comment:
Can you remove the namespace field from the metadata here:
https://github.com/samyak-jn/gitops-service-sample-k8s-resources/blob/52dad3c7a8526031561d4e9820ebc58ba04f7c6d/sample-app/roles.yaml#L5

It appears to be causing the Argo CD application not to be healthy.